GUI/src/main/kotlin/ui/screens/SettingsScreen.kt

```diff
@@ -87,3 +97,14 @@ fun SettingsScreen(state: MutableState<AppState>) {
         }
     }
 }
+
+@Composable
+private fun RowScope.DeleteMaskPath(state: MutableState<AppState>) {
+    Button(
+        onClick = { state.value = state.value.copy(maskPath = null) },
+        modifier = Modifier.Companion.weight(0.1f).fillMaxSize(),
```

Comment:
The FileSelectorButton has a padding which reduces its height, making the two buttons different vertical sizes. Imo, this does not look good. Either, use a padding here too, or make it part of an argument Modifier :)